usaspending_api/idvs/v2/views/amounts.py

```diff
@@ -107,6 +107,53 @@ def fetch_account_details_idv(award_id, award_id_column) -> dict:
     return results
 
 
+def fetch_idv_child_outlays(award_id: int, award_id_column) -> dict:
+    if award_id_column != "award_id":
+        award_id = re.sub(r"[']", r"''", award_id)
+    sql = """
```

Comment:
The SQL doesn't appear to be handling the case of the Award having File C data in the latest period of the FY of its earliest action date. I believe this is handled in your other PR and needs to occur here as well.